### ISSUE TITLE
support iVotronic format

### ISSUE THREAD

**Issue by tarheel** [BrightSpots/rcv#104]
Sample files from George:
- https://www.dropbox.com/l/scl/AAC-5xAQsSJnaBLcoYuyEi-uR1_EZjbflGE (CVR in LST format, whatever that is)
- https://www.dropbox.com/l/scl/AADyOvAwkjNUB1Dxvj8eqQkcDO7LiXC-EsU (CVR in XLSX)
- https://www.dropbox.com/l/scl/AACzSzHBw-XB0jcy-w3LkBLw0-RUrfGnW28 (tabulation in XLSM)

**Comment by CalebKleppner** [comment]
LST format files from ES&S iVotronic are text files that contain all the CVR data, but they are not very helpfully formatted.  I think we should assume we're using the .xlsx form of iVotronic records, which seems to be the same as the ES&S format from Maine and Minneapolis.

**Comment by tarheel** [comment]
@CalebKleppner: given your last comment, should we move this to the "v1.1 Future" milestone?

**Comment by CalebKleppner** [comment]
Yes, definitely v1.1 Future that issue, as with other data formats that 
we don't have good specs for (Dominion, Hart?).  I'll have to look at 
variations on Dominion data, but I think they key places that use it are 
San Francisco and Alameda County (Oakland, Berkeley and San Leandro).

Caleb Kleppner
MK Election Services, LLC
203-376-4080

**Comment by tarheel** [comment]
Is this still something we're going to want to do at some point?
